### DISCUSSION TITLE
Help compiling Bullet with double precision

### DISCUSSION THREAD

**Discussion by santagoy** [bulletphysics/bullet3#4126] [Q&A]
Hello! I'm trying to compile a double precision version of bullet3 for a small project with openmw. Tried altering the OPTION in CMakeLists.txt as seen below:

```OPTION(USE_DOUBLE_PRECISION "Use double precision"	ON)```

That didn't work. I tried the same with the multi-threading to no avail.

I'm on a linux machine fyi and I used the `build_cmake_pybullet_double.sh` file to build it.

Any help would be appreciated! Thank you and I hope to hear from y'all soon.

**Comment by alda30** [comment]
Could you please post the message you get when trying to compile it?
